include/wx/notebook.h

```diff
@@ -139,6 +139,13 @@ class WXDLLIMPEXP_CORE wxNotebookBase : public wxBookCtrlBase
     // new is wxNOT_FOUND)
     void SendPageChangedEvent(int nPageOld, int nPageNew = wxNOT_FOUND);
 
+    // return wxTOP/wxBOTTOM/wxRIGHT/wxLEFT
+    wxDirection GetTabOrientation() const;
+
+    // return the rectangle in window coordinates of the page selector.
+    // For example, for a wxNotebook this would be its tab.
```

Comment:
As also mentioned in the interface file comment, saying "for example for a wxNotebook" in wxNotebook documentation doesn't seem to make much sense, so I'd just write
```suggestion
    // return the rectangle of the given page tab in window coordinates
```